test/test.js

```diff
@@ -14,6 +14,8 @@ var mc = require('../')
   , MC_SERVER_JAR = process.env.MC_SERVER_JAR
   , SURVIVE_TIME = 10000
   , MC_SERVER_PATH = path.join(__dirname, 'server')
+  , getFieldInfo = require('../dist/utils').getFieldInfo
```

Comment:
this is wrong : tests should test only the exposed interface I think (as is done for the rest of the stuff used in test.js), so getFieldInfo and evalCondition should probably be exposed in index.js